packages/static-wado-creator/lib/writer/ImageFrameWriter.js

```diff
@@ -27,19 +27,17 @@ const ImageFrameWriter = (options) => {
         [new MultipartHeader("Content-Type", type.contentType, [new MultipartAttribute("transfer-syntax", transferSyntaxUid)])],
         content
       );
-      writeStream.close();
-      if (verbose) {
-        console.log("Wrote encapsulated image frame", id.sopInstanceUid, index + 1, type.contentType);
-      }
+      await writeStream.close();
```

Comment:
This is the entire fix here - everything else it eslint cleanup (which won't occur on the next commit/push)